tsdb/points.go

```diff
@@ -1309,25 +1308,39 @@ func (p Fields) MarshalBinary() []byte {
 		case int:
 			b = append(b, []byte(strconv.FormatInt(int64(t), 10))...)
 			b = append(b, 'i')
-		case int32:
+		case int8:
 			b = append(b, []byte(strconv.FormatInt(int64(t), 10))...)
 			b = append(b, 'i')
-		case uint64:
-			b = append(b, []byte(strconv.FormatUint(t, 10))...)
+		case int16:
+			b = append(b, []byte(strconv.FormatInt(int64(t), 10))...)
+			b = append(b, 'i')
+		case int32:
+			b = append(b, []byte(strconv.FormatInt(int64(t), 10))...)
 			b = append(b, 'i')
 		case int64:
 			b = append(b, []byte(strconv.FormatInt(t, 10))...)
 			b = append(b, 'i')
+		case uint:
+			b = append(b, []byte(strconv.FormatUint(uint64(t), 10))...)
```

Comment:
Ok, so I think the right fix is to not support uint64.  At this point that using the Go client library if you send a `uint64` in, that it is going to get converted to a string.  If you are using straight line protocol, this won't be an issue, as anything over `int64` will fail one of our checks and error out.

/cc @jwilder @beckettsean